components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/AuthorizationHandlerManager.java

```diff
@@ -95,6 +99,7 @@ public static AuthorizationHandlerManager getInstance() throws IdentityOAuth2Exc
         return instance;
     }
 
+    @Deprecated
```

Comment:
When we add the deprecated annotation, let's mention what method to use instead as well

Reply:
fixed